src/display/api.js

```diff
@@ -185,7 +198,7 @@ function setPDFNetworkStreamFactory(pdfNetworkStreamFactory) {
  * @param {string|TypedArray|DocumentInitParameters|PDFDataRangeTransport} src
  * Can be a url to where a PDF is located, a typed array (Uint8Array)
  * already populated with data or parameter object.
- * @returns {PDFDocumentLoadingTask}
+ * @returns {PDFDocumentLoadingTasks}
```

Comment:
The class is named `PDFDocumentLoadingTask` (and that shouldn't be changed); why did you add a trailing `s` here?